src/stream.c

```diff
@@ -334,6 +334,9 @@ avifBool avifROStreamReadAndEnforceVersion(avifROStream * stream, uint8_t enforc
 #define AVIF_STREAM_BUFFER_INCREMENT (1024 * 1024)
 static avifResult makeRoom(avifRWStream * stream, size_t size)
 {
+    if (size > SIZE_MAX - stream->offset) {
+        return  AVIF_RESULT_OUT_OF_MEMORY;
```

Comment:
Nit: Use only one space character between "return" and "AVIF_RESULT_OUT_OF_MEMORY".